examples/lighting-app/silabs/SiWx917/src/AppTask.cpp

```diff
@@ -229,11 +241,29 @@ void AppTask::LightActionEventHandler(AppEvent * aEvent)
     }
 }
 
+void AppTask::ButtonEventHandler(uint8_t button, uint8_t btnAction)
+{
+    AppEvent button_event           = {};
+    button_event.Type               = AppEvent::kEventType_Button;
+    button_event.ButtonEvent.Action = btnAction;
+    if (button == 1 && btnAction == SL_SIMPLE_BUTTON_PRESSED)
```

Comment:
Same comment as before. Use a define/constant for the button value

Reply:
sure